### PR TITLE
updated some information

### PR DESCRIPTION
**Pull request by Akethorpe**
updated copyright, added windows 8, 8.1, 10, added mac os 10.5-11, changed springtimes format from .ssc to .sm

### CODE REVIEW

File: Manual/index.html [hunk 1]
```diff
@@ -104,7 +104,7 @@ <h4><a name="songinst">Song Installation</a></h4>
 Songs/StepMania 5/Springtime/Kommisar - Springtime.mp3
 Songs/StepMania 5/Springtime/spring.png
 Songs/StepMania 5/Springtime/springbn.png
-Songs/StepMania 5/Springtime/Springtime.ssc
+Songs/StepMania 5/Springtime/Springtime.sm
```
Comment (concubidated) [inline Manual/index.html]: https://github.com/stepmania/stepmania/blob/master/Songs/StepMania%205/Springtime/Springtime.ssc

ssc is the latest and greatest format and we don't even ship this song with a .sm.

**Comment by dougshell** [comment]
Why have you changed file format from .ssc to .SM.

The file is currently distributed with .ssc file type.

Please correct and we can get this pushed to master

**Comment by dguzek** [comment]
For what it's worth, the Xcode project that cmake generates via
 ```
cmake -G Xcode -DWITH_FULL_RELEASE=ON -DCMAKE_BUILD_TYPE=Release ..
```
suggests that (recent) releases of SM5 only support macOS 10.7 and newer.

screen shot 2018-07-25 at 11 48 39 am

I poked around in the `./CMake` directory to try to find where is this specified but I'm too unfamiliar with CMake.

**Comment by Scraticus** [comment]
closing as a 5.2 PR, and it's abandoned

**Pull request closed without merging**